l10n_ch_lsv_dd/model/banking_export_ch_dd.py

```diff
@@ -78,12 +78,11 @@ def create(self, vals):
         readonly=True
     )
     filename = fields.Char(
-        string=_('Filename'),
         size=256,
         readonly=True,
     )
     state = fields.Selection(
-        [('draft', _('Draft')), ('sent', _('Sent'))],
+        [('draft', 'Draft'), ('sent', 'Sent')],

```

Comment:
Ok I checked and this is correct, you can keep it like that ! Good to know for me :wink: